jobs/config.json

```diff
@@ -12992,6 +12992,20 @@
       "sig-testing"
     ]
   },
+  "pull-kubernetes-local-e2e-containerized": {
+    "args": [
+      "--build=bazel",
+      "--deployment=local",
+      "--ginkgo-parallel=1",
+      "--provider=local",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]",
+      "--timeout=120m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-testing"
```

Comment:
I don't think sig-testing owns containerized kubelet. Is this sig-storage and sig-node?